src/consumer/mod.rs

```diff
@@ -91,6 +99,12 @@ pub trait ConsumerContext: ClientContext {
     #[allow(unused_variables)]
     fn pre_rebalance<'a>(&self, rebalance: &Rebalance<'a>) {}
 
+    /// Override this to return true when using cooperative-sticky option for
+    /// partition.assignment.strategy
+    fn is_incremental_assign(&self) -> bool {
```

Comment:
This function can be avoid if there is a way to read the config `partition.assignment.strategy` value

Reply:
Great! So where is the part that read from the config?

Reply:
Thanks. I figured out how to do this automatically.